conf/app.exit.routes

```diff
@@ -5,3 +5,8 @@ GET        /office-of-exit-for-transit/add-another/:lrn               controller
 POST       /office-of-exit-for-transit/add-another/:lrn               controllers.exit.AddAnotherOfficeOfExitController.onSubmit(lrn: LocalReferenceNumber, mode: Mode = NormalMode)
 GET        /office-of-exit/change-add-another/:lrn                    controllers.exit.AddAnotherOfficeOfExitController.onPageLoad(lrn: LocalReferenceNumber, mode: Mode = CheckMode)
 POST       /office-of-exit/change-add-another/:lrn                    controllers.exit.AddAnotherOfficeOfExitController.onSubmit(lrn: LocalReferenceNumber, mode: Mode = CheckMode)
+
+GET        /add-office-of-exit-for-transit/:lrn                       controllers.exit.AddCustomsOfficeOfExitYesNoController.onPageLoad(lrn: LocalReferenceNumber, mode: Mode = NormalMode)
```

Comment:
Not critical but the pattern I've seen most recently with add pages is like: /office-of-exit-for-transit/add/:lrn